tests/resources/test_workflow.py

```diff
@@ -28,7 +28,7 @@ def failed_design_workflow_data(basic_design_workflow_data):
         'status': 'FAILED',
         'status_description': 'ERROR',
         'status_detail': [
-            {'level': 'WARN', 'msg': 'Something is wrong'},
+            {'level': 'WARNING', 'msg': 'Something is wrong'},
```

Comment:
This is still fixing a bug, since the backend does not return the string `WARN`, regardless of casing.